deps/patches/llvm7-AMDGPU-addrspaces.patch

```diff
@@ -0,0 +1,56 @@
+From 86d8f85b0157aa8ce17840adc1121dbc7ae35262 Mon Sep 17 00:00:00 2001
+From: Valentin Churavy <v.churavy@gmail.com>
```

Comment:
Did I really write this patch? I don't recall it and it shouldn't have the Julia specific AS in it.

Reply:
You wrote the initial patch, and then I made tweaks. I can change authorship to myself if you'd prefer.

Reply:
And I believe I added the Julia AS's because there were a few other locations where I needed to handle Julia's addrspaces, although I guess it would just be better to do the `AS > LAST` check you mentioned below. I will make that change.